Detectors/MUON/MCH/Base/src/Digit.cxx

```diff
@@ -19,8 +19,8 @@ bool closeEnough(double x, double y, double eps = 1E-6)
   return std::fabs(x - y) <= eps * std::max(1.0, std::max(std::fabs(x), std::fabs(y)));
 }
 
-Digit::Digit(int detid, int pad, unsigned long adc, Time time, uint16_t nSamples)
-  : mDetID(detid), mPadID(pad), mADC(adc), mTime(time), mNofSamples(nSamples)
+Digit::Digit(int detid, int pad, unsigned long adc, int32_t time, uint16_t nSamples)
+  : mTFtime(time), mDetID(detid), mPadID(pad), mADC(adc), mNofSamples(nSamples)
 {
```

Comment:
[Member variables should be declared and initialized in the same order](https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html?showone=Initialization#Initialization), so `mNofSamples` should be second.

Reply:
Fixed.